include/openssl/crypto.h

```diff
@@ -335,37 +332,34 @@ size_t CRYPTO_secure_used(void);
 void OPENSSL_cleanse(void *ptr, size_t len);
 
 # ifndef OPENSSL_NO_CRYPTO_MDEBUG
+void CRYPTO_get_alloc_counts(int *mcount, int *rcount, int *fcount);
```

Comment:
Huh?  A new function?

Reply:
Nope, just moved from line 348 below...  presumably to keep it separate from the bunch of deprecated ones.  Did I get that right, @richsalz?

Reply:
yes, exactly @levitte :)